src/plugins/DemoApp/pages/BlueprintInteraction.tsx

```diff
@@ -55,8 +56,8 @@ export const BlueprintInteraction = () => {
             defaultDestination={`${dataSourceName}/instances`}
             onCreated={(createdEntity: TReference) =>
               dmssApi
-                .documentGetById({
-                  idReference: `${dataSourceName}/${createdEntity._id}`,
+                .documentGet({
+                  address: `dmss://${dataSourceName}/${createdEntity._id}`,
```

Comment:
```suggestion
                .documentGet({
                  address: `dmss://${dataSourceName}/$${createdEntity._id}`,
```

is there a dollar sign missing before the id here?